### DISCUSSION TITLE
Can I keep the current battery monitor?

### DISCUSSION THREAD

**Discussion by tomasenskede** [Louisvdw/dbus-serialbattery#537] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32595351/232686779-037490cf-9049-4a1c-86ee-d75222f53acd.jpeg)

I just got my Daly USB-UART and are planning the installation between my Daly Smart Stabiliser and Victro Cerbo GX.

While reading I see that this driver act as an battery monitor... I have an BMV-device for SOC-calculation... can I have this device for monitoring the cells within the batteries etc and the BMV for SOC.. 

Thanks for any guidance!

**Comment by mr-manuel** [comment, marked as answer]
Yes, you can have this driver to monitor the cells. Best would be, if you install Venus OS 3.00 since there you have the possibility to also select the BMS. Else your BMS will be reponsible for the DVCC limits (if you have DVCC enabled).

See also https://community.victronenergy.com/questions/199460/multiple-battery-monitor-available-only-selected-a.html?childToView=199503#comment-199503

**Reply by mr-manuel** [reply]
See also the driver documenation.
